docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Ingesting Dynamo DB tables into the Landing Zone
+description: "Ingesting tables from a Dynamo DB instance in an Hakcney AWS account that is attached to the hub into the Data Platform landing zone"
+layout: playbook_js
+tags: [playbook]
+---
+
+## Objective
+
+To setup a job that will ingest specified tables from a Dynamo DB instance.
+The Dynamo DB instance must be in an AWS account that is linked to the Hackney Hub.
```

Comment:
It there a specific reason why it needs to be connected to the Hackney Hub? I was under the impression that DynamoDb doesn't sit within a VPC and therefore doesn't require inter-account connectivity.

Reply:
Ah yeah you're right, I'm getting mixed up here!